api/models.py

```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```

Comment:
You're not using the sorting in the template - you check for the activity level anyway with the if statements (by the way, might be worth using [elif](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#if)).